bootstrap/src/test.rs

```diff
@@ -114,30 +114,36 @@ impl TestCommand {
             let name = format!("bless/{}", filename.to_string_lossy());
             let output_file = manifest.out_dir.join("tests/bless").join(filename);
             let testcase = TestCase::new(name, case, output_file, TestType::Bless, verbose);
-            tests.push(testcase);
+            cases.push(testcase);
         }
 
         // Collect test-auxiliary
-        let aux_use = regex::Regex::new(r"^//@\s*aux-build:(?P<fname>.*)").unwrap();
-        let mut auxiliary = vec![];
-        for case in tests.iter() {
-            let source = std::fs::read_to_string(&case.source).unwrap();
-            for cap in aux_use.captures_iter(&source) {
+        let aux_use = regex::Regex::new(r"\s*//@\s*aux-build:(?P<fname>.*)").unwrap();
```

Comment:
Should probably keep the start anchor, we don't want to accept directives that come after other content

```suggestion
        let aux_use = regex::Regex::new(r"^\s*//@\s*aux-build:(?P<fname>.*)").unwrap();
```

Reply:
Some testcases like `tests/codegen/filename.rs` declare `aux-build` in the third line.

```
//! Test that the generated code has the filename and function name in it

//@ aux-build:mini_core.rs

// CHECK: filename
...
```

Reply:
Ah, the regex should probably start with `(?m)` to enable multi-line mode. Or iterate `.lines()` and check each separately.

Reply:
I solved it.